docs/generated/packages/nx/documents/release.md

```diff
@@ -57,6 +57,114 @@ Show version number
 
 ## Subcommands
 
+### $0
+
+Create a version and release for the workspace, then generate a changelog
+
+```shell
+nx release $0 [specifier]
+```
```

Comment:
The generated docs aren't correct for this case. Any suggestions on how to do a manual override?